app/src/main/java/org/luxoft/sdl_core/CompressionUtil.java

```diff
@@ -0,0 +1,66 @@
+package org.luxoft.sdl_core;
+
+import android.util.Log;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.zip.DataFormatException;
+import java.util.zip.Deflater;
+import java.util.zip.Inflater;
+
+public class CompressionUtil {
+    private static final String TAG = CompressionUtil.class.getSimpleName();
+    private static final int const_buffer_size = 1024;
```

Comment:
@VadymLuchko I think we can improve the compress/decompress speed by setting the buffer size to MTU of SDL protocol
```suggestion
    private static final int const_buffer_size = 131072;
```

Reply:
changed in 89c0984